tests/src/test/fdc3.broadcast.ts

```diff
@@ -792,22 +1008,41 @@ export default () =>
     }
 
     async function wait() {
-      return new Promise((resolve) =>
-        setTimeout(() => {
+      return new Promise((resolve) => {
+        timeout = window.setTimeout(() => {
           resolve(true);
-        }, constants.WaitTime)
-      );
+        }, constants.WaitTime);
+      });
+    }
+
+    async function closeChannelsAppWindow(channelType: ChannelType) {
+      if (channelType === ChannelType.System) {
+        //Join channel 1
+        await window.fdc3.leaveCurrentChannel();
+        await joinChannel(1);
+
+        //Tell channelsApp to close window
+        await broadcastSystemChannelCloseWindow();
```

Comment:
This call is breaking the tests in some cases as its putting context onto the system channel that will be received when a listener is added for it.  These commands must not go over system channels or channels used for testing - instead use a dedicated app channel that is used for nothing else. 

This issue is also in the current version. You can replicate problems by spawning another component (anything) and put it on the first system channel before running the tests. The reason it doesn't fail every tie is that the Finsemble version you are using to test drops context from the channel if everyone leaves (as happens between tests).

Reply:
Will raise an issue